src/components/link-preview/link-preview.d.ts

```diff
@@ -6,6 +6,8 @@ interface ImageShape {
 }
 
 export interface LinkPreviewProps {
+  /** Used to set the root element to either am anchor link or div container */
```

Comment:
```suggestion
  /** Used to set the root element to either an anchor link or div container */
```

Reply:
this file is removed in LinkPreview refactor